src/nccl_ofi_rdma.c

```diff
@@ -1428,7 +1428,18 @@ static inline int process_completions(struct fi_cq_tagged_entry *cq_entry,
 				return ncclInternalError;
 			}
 
-			if (IS_CONN_RESP_MSG_TYPE(cq_entry[comp_idx].tag) && (comp_flags & FI_RECV)) {
+			if (IS_CONN_MSG_TYPE(cq_entry[comp_idx].tag) && (comp_flags & FI_SEND)) {
```

Comment:
Nit: Libfabric doesn't require send completion events to have the tag field set (though EFA does).

Reply:
let's not rely on things that only efa does.